ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/utils/StringUtils.java

```diff
@@ -137,7 +137,7 @@ public static String unescapeUnicodeCodepoints(String identifier) {
      * @return encoded string.
      */
     public static String encodeIdentifier(String string) {
-        return IdentifierUtils.encodeNonFunctionIdentifier(string);
+        return string.replace("\\", "\\\\");
```

Comment:
Will this replacement be enough? I think we may have to test some other cases involving quotes, etc...

Reply:
It seems that this replacement is enough for the test cases that we are currently using (including quotes)